controllers/configuration_controller.go

```diff
@@ -107,9 +130,9 @@ func (r *ConfigurationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 		r.ProviderName = configuration.Spec.ProviderReference.Name
 	}
 	if err := terraformApply(ctx, r.Client, ns, configuration, applyJobName, tfInputConfigMapsName, r.ProviderName); err != nil {
-		return ctrl.Result{RequeueAfter: 3}, errors.Wrap(err, "failed to create/update cloud resource")
+		return ctrl.Result{RequeueAfter: 3 * time.Second}, errors.Wrap(err, "failed to create/update cloud resource")
 	}
-	return ctrl.Result{RequeueAfter: 10}, nil
+	return ctrl.Result{}, nil
```

Comment:
Why is `RequeueAfter` moved?

Reply:
10 is too small to print a large number of logs.
you want 10 * time.Second ?

Reply:
10: 10 nanosecond